.github/workflows/test_definitions.yml

```diff
@@ -0,0 +1,27 @@
+name: Tests
+
+# This uses workflows in a different repo, refer to:
+# https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow
+
+# This runs sailbot workspace workflows and actions.
+# Runs the Tests workflow in sailbot_workspace
+   
+# Run condition
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    # Every Friday at 1:00am UTC
+    - cron: '0 1 * * 5'
+  
+jobs:
+# Calls our workflows in sailbot_workspace 
+# https://github.com/UBCSailbot/sailbot_workspace
+  call-workflow-1-in-sailbot:
+        uses: UBCSailbot/sailbot_workspace/.github/workflows/tests.yml@main
```

Comment:
Need to use this branch until merged into main: `test_definitions.yml@user/colinli02/11-Publish-ROS-package-actions_FINAL`

Reply:
Whoops, thought your previous comment on disabling setting referred to being able to test without @branch. Will change back